### DISCUSSION TITLE
Absolute imports stopped working

### DISCUSSION THREAD

**Discussion by MarianaK96** [vercel/next.js#62148] [Help]
### Summary

I have a next js 14 app where I am using the default absolute import pathing, however it suddenly stopped working and I don't now what has caused it. When this happened before, the only thing that 'fixed' it was starting a new project, but I have done quite a bit of work now and so can't just start over. 

Here is where I am using a component:
```
"use client";

import { Button } from "@/app/components/Button";

const Home = () => {
  return (
    <>
      <Button />
    </>
  );
};

export default Home;

```

this is the component:
```
export const Button = () => {
  return <button>Click me</button>;
};
```

and here is my js config:

```
{
  "compilerOptions": {
    "paths": {
      "@/*": ["./*"]
    }
  }
}

```

**Comment by makibat** [comment]
Since the paths are defined as `"@/*": ["./*"]`, shouldn't you import the component from `@/app` rather than `@app`?

**Reply by MarianaK96** [reply]
I must have sent the wrong code as I was playing around with a few different attempts, but unfortunately that doesn't work either :/
